x/pocketcore/types/proof.go

```diff
@@ -419,7 +422,8 @@ func (c ChallengeProofInvalidData) Bytes() []byte {
 		},
 	})
 	if err != nil {
-		panic(fmt.Sprintf("an error occured converting the challengeproof to bytes\n%v", err))
+		fmt.Println(fmt.Errorf("an error occured converting the challengeproof to bytes\n%v", err))
+		os.Exit(1)
```

Comment:
```suggestion
  log.Fatalf("an error occured converting the relay challengeproof to bytes:\n%v", err)
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)